doc/examples/applications/plot_coins_segmentation.py

```diff
@@ -35,7 +35,9 @@
 
 """
 
-fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(6, 3))
+fig = plt.figure(figsize=(6,3))
+ax1 = fig.add_subplot(1, 2, 1, adjustable='box-forced')
+ax2 = fig.add_subplot(1, 2, 2, sharex = ax1, sharey = ax1, adjustable='box-forced')
```

Comment:
We recently standardized the plot creation API to `fig, (ax0, ax1, ...) = plt.subplots(...)`, which is the preferred Matplotlib style. There are a couple different ways this is implemented in this PR; I prefer the style shown in `plot_morphology.py` to the style here as it continues the use of `plt.subplots`.